src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js

```diff
@@ -13,8 +19,31 @@ export default class App extends Component {
       <Layout>
         <Route exact path='/' component={Home} />
         <Route path='/counter' component={Counter} />
+////#if (!IndividualLocalAuth)
         <Route path='/fetch-data' component={FetchData} />
+////#endif
+////#if (IndividualLocalAuth)
+        <AuthorizeRoute path='/fetch-data' component={FetchData} />
+        <Route path={ApplicationPaths.Login} render={() => loginAction(LoginActions.Login)} />
+        <Route path={ApplicationPaths.LoginFailed} render={() => loginAction(LoginActions.LoginFailed)} />
+        <Route path={ApplicationPaths.LoginCallback} render={() => loginAction(LoginActions.LoginCallback)} />
+        <Route path={ApplicationPaths.Profile} render={() => loginAction(LoginActions.Profile)} />
+        <Route path={ApplicationPaths.Register} render={() => loginAction(LoginActions.Register)} />
+        <Route path={ApplicationPaths.LogOut} render={() => logoutAction(LogoutActions.Logout)} />
+        <Route path={ApplicationPaths.LogOutCallback} render={() => logoutAction(LogoutActions.LogoutCallback)} />
+        <Route path={ApplicationPaths.LoggedOut} render={() => logoutAction(LogoutActions.LoggedOut)} />
```

Comment:
These routes are quite a lot to copy in here. Since all these paths and actions are managed inside the `api-authorization` code, could all these auth-specific routes be wrapped up in a child component that lives inside `api-authorization`? Example: https://stackoverflow.com/a/43311025

Reply:
I tried this but I didn't manage it to get it working. I've filed an issue to give it another spin after preview3.